### PR TITLE
restruct

### PR DESCRIPTION
**Pull request by Tianhao-Gu**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -15,7 +15,7 @@ COPY ./ /kb/collections/
 
 # Write the git commit for the service
 ARG VCS_REF=no_git_commit_passed_to_build
-RUN echo "GIT_COMMIT=\"$VCS_REF\"" > common/git_commit.py
+RUN echo "GIT_COMMIT=\"$VCS_REF\"" > src/common/git_commit.py
```
Comment (MrCreosote) [inline Dockerfile:18]: The entrypoint command will need an update as well

File: src/service/app.py [hunk 2]
```diff
@@ -3,8 +3,8 @@
 '''
 import os
 
-from common.git_commit import GIT_COMMIT
-from common.version import VERSION
+from src.common.git_commit import GIT_COMMIT
+from src.common.version import VERSION
```
Comment (MrCreosote) [inline src/service/app.py:7]: I'm not sure whether the common python practice is to include src in the import path or not. If not, we'd need to set the PYTHONPATH to /src. 

Personally I don't really care either way as long as we're consistent.

**Review by MrCreosote — CHANGES_REQUESTED** [review]
Just needs a docker file fix

File: Dockerfile (hunk 1 above)
Comment (Tianhao-Gu) [reply Dockerfile:18]: 👍

**Pull request merged**